ch.elexis.core.ui/src/ch/elexis/core/ui/views/FallDetailBlatt2.java

```diff
@@ -566,7 +576,7 @@ public void setFall(final Fall f){
 			ix = 0;
 		}
 		cReason.select(ix);
-		String abr = f.getAbrechnungsSystem();
+		
```

Comment:
Das darf aber nicht sein oder? das abr wird ja in der nächsten Zeile verwendet?!